acceptance_tests/features/environment.py

```diff
@@ -64,6 +66,16 @@
 
 
 def before_all(_):
+    # Clear up old screenshots if in screenshot mode
+    if os.getenv('SCREENSHOT', 'False') == 'True':
+        try:
+            os.unlink(os.path.join(get_screenshot_directory(), '*_failed.png'))
+        except FileNotFoundError:
+            # ignore if there were no previous screenshots to delete
+            print()
```

Comment:
For anyone wondering, this is essentially a `noop` because `flake8` needs an `except` block to contain something, whereas this one is telling it to specifically _ignore_ errors if no files are found.

Note that only `FileNotFoundError` is actually excepted - all other errors will still raise, albeit in a slightly roundabout way.

The generic `except` is functionally identical to not having it, but acts as a verbosity improvement to highlight that the code is raising all but `FileNotFoundError`.

Reply:
In python you can also use a 
```
with suppress(FileNotFoundError):
        code here
```
which i think could work. Not to sure of doing it this way though

Reply:
^ Ignore that comment won't work while looking for the other exception

Reply:
Have changed it to a pass.  Occurred to me that could also just have a check in a generic except block to check if it was a `FileNotFound`